scripts/document.sh

```diff
@@ -51,8 +43,19 @@ jazzy \
     --theme ${THEME} \
     --output ${OUTPUT} \
     --build-tool-arguments CODE_SIGN_IDENTITY=,CODE_SIGNING_REQUIRED=NO,CODE_SIGNING_ALLOWED=NO
+    
+# Link to turf documentation
+TURF_VERSION=$(grep -A 5 Turf Package.resolved | grep version | cut -d '"' -f 4)
```

Comment:
Not sure if it's easier to read and understand, but
```suggestion
TURF_VERSION=$(cat Package.resolved | python -c 'import sys, json; package = next(p for p in json.load(sys.stdin)["object"]["pins"] if p["package"] == "Turf"); print(package["state"]["version"])')
```

Reply:
By the way, if we’re worried about the shell scripting approach being too fragile, we could use a tool like [`jq`](https://stedolan.github.io/jq/), which works with JSON in a shell-scripty manner. Python could work too, but it would strongly suggest rewriting the whole script in a more understandable language. 😅

Reply:
Went with a python oneliner not to introduce new dependecies.